### PR TITLE
Create a getting started guide

### PR DESCRIPTION
**Pull request by tylerphelan**
https://github.com/pivotal-cf/docs-build-service/issues/173

### CODE REVIEW

File: getting-started.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+This page is meant to serve as a quick-start guide and may not include some configurations required for your specific environment. For more details on installation, see [Installing Tanzu Build Service](installing.html).
+
+## <a id='prerequisites'></a> Prerequisites
+
+Before you install Build Service, you must:
+
+* Have access to the Kubernetes cluster satisfying the [minimum required permissions](#install-rbac).
+
+* Ensure your Kubernetes cluster is configured with default `StorageClass`.
```
Comment (matthewmcnew) [inline getting-started.html.md.erb:16]:
```suggestion
* Ensure your Kubernetes cluster is configured with a default `StorageClass`.
```

**Comment by tylerphelan** [comment]
This includes `kp ... --ca-cert-path` which is not released, I'll merge this as soon as it is.

File: getting-started.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+1. Create a Kubernetes Secret that will allow your Builds to push to the desired registry with the `kp` cli:
+
+```
+kp secret create my-registry-creds --registry my.registry.io --registry-user my-user
+```
+
+You will be prompted for your password (`my-password`).
+
+3. Create the Tanzu Build Service Image:
```
Comment (sukhil-suresh) [inline getting-started.html.md.erb:150]: numbering is off I think

Comment (sukhil-suresh) [inline getting-started.html.md.erb:150]: numbering is off I think

File: getting-started.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+```
+ytt -f /tmp/values.yaml \
+    -f /tmp/manifests/ \
+    -f /tmp/ca.crt \
+	-v docker_repository="my.registry.io/tbs" \
+	-v docker_username="my-user" \
+	-v docker_password="my-password" \
+	| kbld -f /tmp/images-relocated.lock -f- \
+	| kapp deploy -a tanzu-build-service -f- -y
```
Comment (sukhil-suresh) [inline getting-started.html.md.erb:89]: been wondering recently, if it makes sense for us to skip the auto confirmation `-y` bit for the the `kapp deploy`. I think the summary is skipped if we do the `-y`? can't remember. Could always mention about the `-y` flag for scripting purpose.

Comment (sukhil-suresh) [reply getting-started.html.md.erb:89]: so much more easier to review all this rendered

Comment (matthewmcnew) [reply getting-started.html.md.erb:89]: That is an interesting question.

**Review by matthewmcnew — APPROVED** [review]

**Review by sukhil-suresh — APPROVED** [review]

**Review by mgibson1121 — APPROVED** [review]

**Pull request merged**